adcm_aio_client/core/objects/cm.py

```diff
@@ -26,6 +26,17 @@ class ADCMEntityStatus(str, Enum):
     DOWN = "down"
 
 
+class ADCM(InteractiveObject, WithActions, WithConfig):
```

Comment:
`InteractiveObject` has `id` property by default, let's override it in here and always return like

```python

@property
def id(self: Self) -> int:
     return 1
```